tdp_vars_defaults/loki/loki_promtail.yml

```diff
@@ -30,14 +33,49 @@ promtail_configuration:
     filename: "{{ promtail_data_dir }}/positions.yaml"
   clients:
     - url: "https://{{ loki_server }}:3100/loki/api/v1/push"
-  scrape_configs:
-  - job_name: system
-    static_configs:
-    - targets:
-        - localhost
-      labels:
-        job: varlogs
-        __path__: /var/log/*log
 
-# promtail service start on boot policies
-promtail_start_on_boot: false
+promtail_scrape_configs: []
+#
+# For each entry below, if hosts belongs to one of the groups listed,
+# a scrape config will be added in promtail configuration file
+# default scraped files will be *.logs inside <xxx_log_dir>.
+# - this default directory may be overriden by specifying a 'directory' value
+# - default file patern may be overriden by specifying a 'files' value
+#
+promtail_log_jobs:
+  hadoop: 
+    groups: [ 'hadoop_client' , 'hadoop_master' ]
+  hbase: 
+    groups: [ 'hbase_client', 'hbase_master', 'hbase_rest', 'hbase_rs']
+  hdfs: 
+    groups: [ 'hdfs_dn', 'hdfs_httpfs', 'hdfs_jn', 'hdfs_nn' ]
+  hive: 
+    groups: [ 'hive_client', 'hive_ms', 'hive_s2']
+  hue: 
+    groups: [ 'hue_server' ]
+    directory: '/var/log/hue'
+  jupiterhub: 
+    groups: [ 'jupyterhub_edge', 'jupyterhub_server' ]
+  kafka: 
+    groups: [ 'kafka_broker', 'kafka_client' ]
+  knox: 
+    groups: [ 'knox' ]
+  livy: 
+    groups: [ 'livy_server', 'livy_spark_server' ]
+  mapred: 
+    groups: [ 'mapred_jhs']
+  phoenix: 
+    groups: [ 'phoenix_queryserver_client', 'phoenix_queryserver_daemon']
+  postgresql: 
```

Comment:
PostgreSQL is not a component of TDP, should we add it to ```promtail``` configuration for the moment?

Note:
Postgres' logs are not displayed on Grafana dashboard, because of permission issue for ```promtail``` user:
```drwx------. 2 postgres postgres [...] /var/log/postgresql/```